Resources/config/security.xml

```diff
@@ -41,7 +41,9 @@
 		</service>
 		<service id="scheb_two_factor.provider_registry" class="%scheb_two_factor.provider_registry.class%" lazy="true">
 			<argument type="service" id="scheb_two_factor.session_flag_manager" />
-			<argument /> <!-- Two-factor providers -->
+            <argument type="service" id="event_dispatcher" />
+            <argument type="string" id="%auth_code%" />
+            <argument /> <!-- Two-factor providers -->
```

Comment:
If you shift the providers argument to another position - you need to change the compiler pass that relies on it https://github.com/scheb/two-factor-bundle/blob/master/DependencyInjection/Compiler/ProviderCompilerPass.php#L36